diff --git a/src/executors/add/add.impl.js b/src/executors/add/add.impl.js
index a6cb5d2f399d917b29dc7231dd14e75d7b7694c3..dce5429d7c16eff1efe258f657f4c2d057c2756a 100644
--- a/src/executors/add/add.impl.js
+++ b/src/executors/add/add.impl.js
@@ -4,16 +4,11 @@ exports.addExecutor = void 0;
 const tslib_1 = require("tslib");
 const core_1 = require("@nx-extend/core");
 function addExecutor(options, context) {
-    return tslib_1.__awaiter(this, void 0, void 0, function* () {
-        const { root } = context.workspace.projects[context.projectName];
-        return (0, core_1.execCommand)((0, core_1.buildCommand)([
-            'npx shadcn-ui@latest add',
-            options.component,
-            options.overwrite && '--overwrite',
-            '--path=src',
-            `--cwd=${root}`
-        ]), {});
-    });
+  return tslib_1.__awaiter(this, void 0, void 0, function* () {
+    const { root } = context.workspace.projects[context.projectName];
+    return (0,
+    core_1.execCommand)((0, core_1.buildCommand)(["pnpm dlx shadcn-ui@latest add", options.component, options.overwrite && "--overwrite", "--path=src", `--cwd=${root}`]), {});
+  });
 }
 exports.addExecutor = addExecutor;
 exports.default = addExecutor;
